gitium/inc/class-gitium-submenu-status.php

```diff
@@ -195,13 +243,14 @@ private function show_git_changes_table( $changes = '' ) {
 	}
 
 	private function show_git_changes_table_submit_buttons( $changes ) {
+		$behind_commits = count( $this->git->get_behind_commits());
 		if ( ! empty( $changes ) ) : ?>
 			<p>
-			<label for="save-changes"><?php _e( 'Commit message', 'gitium' ); ?>:</label>
-			<input type="text" name="commitmsg" id="save-changes" class="widefat" value="" placeholder="<?php printf( __( 'Merged changes from %s on %s', 'gitium' ), get_site_url(), date( 'm.d.Y' ) ); ?>" />
+				<label for="save-changes"><?php _e( 'Commit message', 'gitium' ); ?>:</label>
+				<input type="text" name="commitmsg" id="save-changes" class="widefat" value="" placeholder="<?php printf( __( 'Merged changes from %s on %s', 'gitium' ), get_site_url(), date( 'm.d.Y' ) ); ?>" />
 			</p>
 			<p>
-			<input type="submit" name="GitiumSubmitSaveChanges" class="button-primary button" value="<?php _e( 'Save changes', 'gitium' ); ?>" <?php if ( get_transient( 'gitium_remote_disconnected' ) ) { echo 'disabled="disabled" '; } ?>/>&nbsp;
+				<input type="submit" name="GitiumSubmitSaveChanges" class="button-primary button" value="<?php _e( 'Save changes', 'gitium' ); ?>" <?php if ( get_transient( 'gitium_remote_disconnected' ) ) { echo 'disabled="disabled" '; } ?>/>&nbsp;
```

Comment:
```php
$button_value = _e( 'Save changes', 'gitium' );
if ( count( $this->git->get_behind_commits())) {
    $button_value = _e( 'Pull & Save changes', 'gitium' );
}

```

There are 3 cases:
 * remote changes -> Pull Changes
 * remote & local -> Pull & Push Changes
 * only local -> Push Changes


```suggestion
				<input type="submit" name="GitiumSubmitSaveChanges" class="button-primary button" value="<?php $button_value; ?>" <?php if ( get_transient( 'gitium_remote_disconnected' ) ) { echo 'disabled="disabled" '; } ?>/>&nbsp;
```